### ISSUE TITLE
Broken Links

### ISSUE THREAD

**Issue by shahbazsyed** [ray1007/pytorch-word2vec#1]
Hi,
The links in the Related work section for Tensorflow tutorial and implementations are broken.

**Comment by ray1007** [comment]
You're right. The readme is still in progress now. Was busy fixing some bug these days. I'll keep this issue open until the readme is done, and thanks for pointing that out! :)

To be honest, I didn't expect someone to notice this repo this soon :p. 

Tzu-Ray
